pkg/updater/read.go

```diff
@@ -63,14 +63,15 @@ func gcsColumnReader(client gcs.Client, buildTimeout time.Duration, concurrency
 		}
 		log.WithField("total", len(builds)).Debug("Listed builds")
 
-		return readColumns(ctx, client, log, tg, builds, stop, buildTimeout, receivers)
+		readColumns(ctx, client, log, tg, builds, stop, buildTimeout, receivers)
+		return nil
 	}
 }
 
 // readColumns will list, download and process builds into inflatedColumns.
-func readColumns(ctx context.Context, client gcs.Downloader, log logrus.FieldLogger, group *configpb.TestGroup, builds []gcs.Build, stop time.Time, buildTimeout time.Duration, receivers chan<- InflatedColumn) error {
+func readColumns(ctx context.Context, client gcs.Downloader, log logrus.FieldLogger, group *configpb.TestGroup, builds []gcs.Build, stop time.Time, buildTimeout time.Duration, receivers chan<- InflatedColumn) {
 	if len(builds) == 0 {
```

Comment:
Note: this does make the case where len(builds) == 0 go from "gcsColumReader returns an error" to "silently do nothing". I think the latter is probably better behavior; feel free to unhold if you agree.

Reply:
It wouldn't return an error in this situation previously though (it returns nil when there are no builds)